stable/pod-gpu-metrics-exporter/requirements.yaml

```diff
@@ -0,0 +1,9 @@
+dependencies:
+
+- name: node-feature-discovery
+  version: 0.1.0
+  repository: https://v3io.github.io/helm-charts/stable
+
+- name: gpu-feature-discovery
+  version: 0.1.0
+  repository: https://v3io.github.io/helm-charts/stable
```

Comment:
Listing these as requirements seems ok IMHO for now, but we may need to disassociate those and make the mode and GPU discovery a separate "iguazio service" in the future